src/test/fixtures/plugin-kotlin-mpp-report-dir/expected/build/my-reports/licensee/js/artifacts.json

```diff
@@ -22,5 +22,37 @@
                 "url": "https://www.apache.org/licenses/LICENSE-2.0"
             }
         ]
+    },
+    {
+        "groupId": "org.jetbrains.kotlin",
+        "artifactId": "kotlin-dom-api-compat",
+        "version": "1.9.0",
+        "name": "Kotlin Dom Api Compat",
+        "spdxLicenses": [
+            {
+                "identifier": "Apache-2.0",
+                "name": "Apache License 2.0",
+                "url": "https://www.apache.org/licenses/LICENSE-2.0"
+            }
+        ],
+        "scm": {
+            "url": "https://github.com/JetBrains/kotlin"
+        }
+    },
+    {
+        "groupId": "org.jetbrains.kotlin",
```

Comment:
But missing here using Kotlin MPP plugin. I think, it should be able added to a common report, which does not exist yet.

Reply:
Oh, it is missing in Kotlin JVM builds too: https://github.com/cashapp/licensee/blob/trunk/src/test/fixtures/plugin-kotlin-jvm/expected/build/reports/licensee/artifacts.json